docs/products/kafka/kafka-connect/howto/enable-automatic-restart.rst

```diff
@@ -0,0 +1,30 @@
+Enable Apache Kafka Connect connectors auto restart on failures
+===============================================================
+
+In case of an Apache Kafka Connect connector failure, automatic task restart is generally not recommended. Proper investigation on the problem's cause should be performed before attempting to restart the failed connector to avoid experiencing similar problems in the future. However, sometimes a task can fail due to a rare problem like the dedicated node going out of memory due to a huge surge of data. 
```

Comment:
I wonder if active voice is better than to have passive in this paragraph. It is a bit heavy to read right now.

If you have a failure in a Kafka connector, instead of immediately restarting it we recommend a proper investigation of the cause so that similar problems are avoided in the future. 

However....

Reply:
Agreed, thanks!